content/kubermatic/main/tutorials-howtos/encryption-at-rest/enable-encryption-at-rest/_index.en.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Enable or Disable Encryption At Rest"
```

Comment:
I think we should change the title to "Configure via Dashboard" since this page is mostly covering the dashboard configuration.